models/orderDao.js

```diff
@@ -0,0 +1,176 @@
+const { appDataSource } = require("./appDataSource");
+
+const queryRunner = appDataSource.createQueryRunner();
+
+const OrderStatusId = Object.freeze({
+  orderDone: 1,
```

Comment:
```suggestion
  ORDER_DONE: 1,
```
enum value는 전부 `위와같은 형태(ALL_CAPS)`로 네이밍해주시면 좋을 것 같습니다!